content/rule_book/material_and_questions.md

```diff
@@ -86,6 +86,12 @@ Question Type maximums can be exceeded in Overtime.
     - The identification of a singular pronoun does not meet the above requirement.
     - The identification of a plural pronoun does meet the above requirement.
 - If one of the listed answers to the multiple answer's questions is a "negative" answer, this is deemed invalid as a Multiple Answer question.
+    - A "negative" answer is an answer that begins with the words or words similar to "not", "no", or "neither" or are used to contrast another answer.
```

Comment:
I'm totally fine with this, but wonder if it's not FE enough so as to make ratification incrementally tougher.

Reply:
I think we're in the same boat. I think it definitely makes things clearer and is a useful definition, but does that definition change anything about how it was interpreted before?

Reply:
yeah, so here's the thing. It's very likely aside from Jeremy/Gryphon/Me, literally no one has thought/interpreted/realized that "if the question is not answered" COULD apply to scenarios other than "not/no/neither"

so in reality, your changes likely don't change anyone else's interpretation.

Reply:
We are such total nerds.

Reply:
"Unless you give up your cross and follow me, you can never...."
I think the question "You can never ...., why?" Is invalid, because the question is not answered. "Unless" qualifies for me. 

But again, likely no one else has thought of this. 

It was Jeremy that first brought up these "other" "negative" grammatical occurrences.

Reply:
So would you suggest that everything about the negative answer be removed as it is excluded elsewhere in a more generic setting, not just multiples?

Reply:
Or would you only remove the definition? (and examples)

Reply:
"Most" FE
* Leave everything as it is. 

Slightly less FE
* Accept the changes in your PR to the MA definition

Ideal, not FE at all, not v1 at all
* Remove everything regarding pos/neg MAs altogether, AND the Invalid Questions section on "if the question is not answered"

Slightly less ideal, not FE at all, not v1 at all
* Remove the Invalid Questions section on "if the question is not answered"
* Add something to the MA defs, Zach's wording probably works fine.
This is what I added in an earlier PR, but was removed after feedback: "If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question."
![Screen Shot 2020-11-17 at 1 22 36 PM](https://user-images.githubusercontent.com/13351944/99430954-fea15e00-28d7-11eb-806f-b29b4f49ff2d.png)

Reply:
With your version, then negative answer is still undefined. Does it need defining?

It also allows for a list of all negative answers because of the AND statement. While the original wording also uses and, it comes across slightly differently to me, and I'm not sure why. It reads very much as "this, not this" would not be valid, since it acts as a sort of clarification. And since other clarifications are not allowed as MAs, it also isn't allowed in this sort of a scenario.

Reply:
Up to you guys, but I'd vote for the "Slightly less FE" option. I think it's "close enough to FE" that other than the 4 of us, hardly anybody in Quizzingdom will notice or care if they did notice.

Reply:
I guess defining negative answer could still be FE. The only definition I think should be written at this stage is "a negative answer is when the question is not answered" or something that basically just uses the language from Invalid Questions. 

I think multiple negative answers are intended to be invalid. 

You're right that the initial language definitely reads like "this, not that"

But I think that interpretation is inconsistent with the Invalid Questions language. 

So either we consider the current a rulebook bug and make our best guess. Or simply define "negative" using the language from Invalid Questions, and pursue a larger overhaul during v-next.

Reply:
What about just adding the language from Invalid questions to the MA section as a further definition and then using the not/nor/neither as a refined definition of not answering the question.

- If one of the listed answers to the multiple answer's questions is a "negative" answer, this is deemed invalid as a Multiple Answer question.
    - A "negative" answer are answers used to contrast another answer, or does not answer the question (e.g. An answer that begins with the words or words similar to "not" or "neither".)

Reply:
I don't think that's FE enough. Someone clearly wanted "if the question isn't answered" under Invalid Questions, not just specific to MAs.

Reply:
I would still leave it under Invalid questions, but then refine it further under MAs to highlight the words that make it negative specifically in the case of MAs. 

I just checked and this rule under Invalid questions was added in 2017 as was the section under MA. Neither were present in the 2013 version. 

Personally I do think negative answers can answer a question, so I'd be curious to learn why it was added in 2017 under Invalid questions. Negative answers under MA makes a little more sense as I can see it being clarification if used to contrast the other answer, which is how I read the 2018 rulebook version of this rule.

Reply:
> I would still leave it under Invalid questions, but then refine it further under MAs to highlight the words that make it negative specifically in the case of MAs.

I think this is FE and makes sense to me.


> Personally I do think negative answers can answer a question
As do I. But clearly the proponent and/or writer of this rule don't think so.


> I'd be curious to learn why it was added in 2017 under Invalid questions.
Me too


> Negative answers under MA makes a little more sense as I can see it being clarification if used to contrast the other answer
Agreed.


> which is how I read the 2018 rulebook version of this rule.
Was there a 2018 version of this rule, different from the 2017 version?

I just know I read the stuff under MAs, and was like "fine, it's what Karen wanted" and then found the bit under Invalid Questions and was like "heeeyyooooo, what now???"

Reply:
1) I agree as well. If we're all in agreement then I will make that change and then it should be ready to merge. 
2) The content was the same between 2017 & 2018, just the formatting changed, I did a copy edit to make sure sections didn't have a single line on another page etc.  Literally word for word the same. The version we are striking through is the 2018 version